packages/flame/lib/src/components/text_box_component.dart

```diff
@@ -210,17 +216,18 @@ class TextBoxComponent<T extends TextRenderer> extends PositionComponent {
     _textRenderer.render(c, line, Vector2(_boxConfig.margins.left, dy));
   }
 
-  void redrawLater() async {
-    _cache = await _redrawCache();
+  Future<void> redraw() async {
+    final newSize = _recomputeSize();
```

Comment:
I noticed before we updated the size before actually finishing redrawing, I figure this is slightly better like so though it hardly makes any practical difference